src/Layout.jsx

```diff
@@ -9,12 +9,14 @@ var Row = ReactBootstrap.Row;
 var HeaderLinks = require('./HeaderLinks.jsx');
 var NetworkErrorDialog = require('./NetworkErrorDialog.jsx');
 var NewContactDialog = require('./NewContactDialog.jsx');
+var PermissionDialog = require('./PermissionDialog.jsx');
 var Footer = require('./Footer.jsx');
 
 module.exports = React.createClass({
     getInitialState() {
         return {
             newContact: false,
+            permissions: false,
```

Comment:
Can these dialogs be visible both at once? If not would be nice to use some kind of enum (`visibleDialog = Dialog.NoDialog | Dialog.NewContact | Dialog.Permissions`).

Reply:
Good suggestion. There can only be one modal dialog at a time.